### ISSUE TITLE
Add support for sending region and device type in join/call requests to locus

### ISSUE THREAD

**Issue by rimarsh** [ciscospark/spark-ios-sdk#29]
In order to support decoupling Locus and WDM, clients should include the same deviceType and region information with join/call requests as they do in WDM registration.

Today the join request looks like:
```
{
    "correlationId": "bc5a6d40-9f7f-4691-8186-cf555a999883",
    "deviceUrl": "https://wdm-a.wbx2.com/wdm/api/v1/devices/ios/80f3cea3-c6ab-473e-9de2-95c58cc8d5cf",
    "localMedias": [ {
        "localSdp": "<sdp value>",
        "topologyHints": [],
        "type": "SDP",
    } ],
    "moderator": false,
    "supportsNativeLobby": true
}
```

Change it to:
```
{
    "correlationId": "bc5a6d40-9f7f-4691-8186-cf555a999883",
    "localMedias": [ {
        "localSdp": "<sdp value>",
        "topologyHints": [],
        "type": "SDP",
    } ],
    "device": {
        "url": "https://wdm-a.wbx2.com/wdm/api/v1/devices/ios/80f3cea3-c6ab-473e-9de2-95c58cc8d5cf",
        "deviceType": “xxx”,
        "regionCode": “US-WEST”,
        "countryCode: “01”
    },
    "moderator": false,
    "supportsNativeLobby": true
}
```

where region and country code come from `https://ds.ciscospark.com/v1/region`.

**Comment by kliu** [comment]
@rimarsh Thank you very much for the updates. We will do it ASAP.

**Comment by rimarsh** [comment]
@kliu, thanks. I've since learned this api isn't ready, so please put it on hold.

**Comment by rimarsh** [comment]
@kliu, this should be working now.

**Comment by kliu** [comment]
It was implemented in 1.3 release.
